pkg/mons/mon-endpoints.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package mons
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+
+	"github.com/kubectl-rook-ceph/pkg/k8sutil"
+	log "github.com/sirupsen/logrus"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const monConfigMap = "rook-ceph-mon-endpoints"
+
+var script = `
+echo %s  | tr -d '=' | sed 's/[A-Za-z]*//g'
+`
+
+func GetMonEndpoint(clusterNamespace string) {
+	// Get Kubernetes Client
+	_, client := k8sutil.GetKubeClient()
+
+	monCm, err := client.ConfigMaps(clusterNamespace).Get(context.TODO(), monConfigMap, v1.GetOptions{})
+	if err != nil {
+		log.Errorf("failed to get mon configmap %s %v", monConfigMap, err)
+	}
+
+	monData := monCm.Data["data"]
+
+	c := exec.Command("/bin/bash",
```

Comment:
Instead of bringing in the bash parsing, could we using go methods to parse instead? Methods like `strings.Split()` are fairly simple and then could be easily unit tested as well.

Reply:
done, using regex